scripts/release.sh

```diff
@@ -237,14 +258,24 @@ QUAY_OWNER=${QUAY_OWNER:="microshift"}
 API_DATA="$(generate_api_release_request "true")" # leave body empty for now
 
 IMAGE_REPO="quay.io/$QUAY_OWNER/microshift"
+AIO_IMAGE_REPO="quay.io/$QUAY_OWNER/microshift-aio"
 RELEASE_IMAGE_TAGS=("$IMAGE_REPO:$VERSION-linux-amd64" "$IMAGE_REPO:$VERSION-linux-arm64" )
+AIO_RELEASE_IMAGE_TAGS=("$AIO_IMAGE_REPO:$VERSION-linux-amd64" "$AIO_IMAGE_REPO:$VERSION-linux-arm64" )
 
 STAGING_DIR="$ROOT/_output/staging"
 mkdir -p "$STAGING_DIR"
 
-build_container_images_artifacts                                      || exit 1
-STAGE_DIR=$(stage_release_image_binaries)                             || exit 1
-push_container_image_artifacts                                        || exit 1
-push_container_manifest                                               || exit 1
-UPLOAD_URL="$(git_create_release "$API_DATA" "$TOKEN")"               || exit 1
-git_post_artifacts "$STAGE_DIR" "$UPLOAD_URL" "$TOKEN"                || exit 1
+# publish containerized microshift 
+build_container_images_artifacts                                          || exit 1
+STAGE_DIR=$(stage_release_image_binaries)                                 || exit 1
+push_container_image_artifacts                                            || exit 1
+push_container_manifest "$IMAGE_REPO" "${RELEASE_IMAGE_TAGS[@]}"          || exit 1
+
+# publish binaries 
+UPLOAD_URL="$(git_create_release "$API_DATA" "$TOKEN")"                   || exit 1
+git_post_artifacts "$STAGE_DIR" "$UPLOAD_URL" "$TOKEN"                    || exit 1
+
+# publish aio container 
+build_aio_container_images_artifacts                                      || exit 1
+push_aio_container_image_artifacts                                        || exit 1
+push_container_manifest "$AIO_IMAGE_REPO" "${AIO_RELEASE_IMAGE_TAGS[@]}"  || exit 1
```

Comment:
The only changes here are to space out the existing steps into logical groupings, without mutating the order, and to add the AIO steps last.  Added parameter passing to functions that were made generic above.